src/test/java/com/artipie/api/AuthRestOrgTest.java

```diff
@@ -217,12 +260,22 @@ public Authentication authentication() {
 
             @Override
             public Policy<?> policy() {
-                return Policy.FREE;
+                final BlockingStorage asto = new BlockingStorage(AuthRestOrgTest.super.ssto);
+                asto.save(
+                    new Key.From(String.format("users/%s.yaml", AuthRestOrgTest.NAME)),
+                    String.join(
+                        "\n",
+                        "permissions:",
+                        "  adapter_all_permission: {}"
```

Comment:
We need to rename `adapter_all_permission` to `all_permissions`

Reply:
ok, first, we need to correct `http`